### ISSUE TITLE
cannot download and execute a model locally

### ISSUE THREAD

**Issue by zfarrell13** [facebookresearch/demucs#555]
I have the appropriate models for htdemucs_ft downloaded and in a folder:

```
/models
     /92cfc3b6-ef3bcb9c.th
     /04573f0d-f3cf25b2.th
     /d12395a8-e57c48e6.th
     /f7e0c4bc-ba3fe64a.th
```
I am trying to execute them locally with the command:

`python3 -m demucs.separate -o separated_sources --repo .../models -n htdemucs_ft --mp3 --mp3-bitrate=320`

but i keep getting the error:

```
FATAL: htdemucs_ft is neither a single pre-trained model or a bag of models.
Command failed, something went wrong.
```
I see repo.py is responsible for handling --repo arguments via the command line.  is there any specific structure that the models must be saved in?

could anyone assist please?

**Comment by zfarrell13** [comment]
for anyone else that will be trying this, you can figure out which models you need using the collab on the readme, defining the model name in the "model =" parameter.  download all models locally from the links, you also need the yaml file associated to the model name in the directory you place all of the downloaded models into.  after that a --repo argument followed my the model directory can be run via the command line with the other model arguments in order to run them locally.  for anyone that had a hard time or is not seasoned in reading the raw code to figure this out, feel free to msg me.

**Comment by zfarrell13** [comment]
closing

**Comment by berkanz** [comment]
@zfarrell13  Have the same problem. I try to load mdx locally. I downloaded all checkpoints in the model bag as well as the [mdx.yaml](https://github.com/facebookresearch/demucs/blob/bd98f01aac5a9e1b1db55cb5744c1ffa69ff39e1/demucs/remote/mdx.yaml) in to a folder and tried to run the python command: 
`demucs.separate.main(["--repo", "mdx/", "-n", "mdx", [local_fn])` but the problem persists.

I realized that the checkpoint names weren't exactly matching, the downloaded ones had a tailing string (i.e., **downloaded one**: c511e2ab-fe698775.th, **written in yaml**: c511e2ab.th). I removed the tailing strings to match them but it didn't help. What am I missing?
